src/System.Net.Http/tests/FunctionalTests/XUnitAssemblyAttributes.cs

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+// [ActiveIssue(35002)]: Disabling parallel execution of System.Net.Http.FunctionalTests
```

Comment:
nit: you should modify the comment like this:

"Disable parallel execution of System.Net.Http.FunctionalTests on UAP test runs"

This way it is obvious if someone looks at this file they will know that this behavior is only for UAP test runs.  Otherwise, they will have to look at the CSPROJ file.

Reply:
Will fix.